src/runner.ts

```diff
@@ -56,8 +58,25 @@ export class Runner {
 	}
 
 	public async run() {
+		const EXTENSION_LINK_START = terminalLink(
+			'Click to view the live results of this run in the Intuita VSCode Extension!',
+			`vscode://intuita.intuita-vscode-extension/case/${this.__caseHashDigest}`,
+		);
+
+		const EXTENSION_LINK_END = terminalLink(
```

Comment:
why do people care about the view being live or not. lets keep things simple and concise. also, is it both for run and dry-run?

here is my suggestion (you can change it in a separate PR. low pri):

"Click here to view the results of this run in the Intuita VSCode Extension. 
(Features include familiar diff view, tweaking & managing the changes, reporting issues, and many more.)"

@r4zendev @grzpab